{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -8,11 +8,10 @@ before_install:
     - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
 install:
+    - make ci_requirements
     - pip install -U pip wheel codecov
-    - make requirements
 script:
     - make validate_translations
     - make validate
-    - make pii_check
```

Comment:
Note for reviewers: PII checking isn't removed; it's just been rolled into `make validate`